File: Plugins/Aspose_Cells_Java_for_PHP/src/aspose/cells/WorkingWithWorksheets/SecurityFeatures/UnprotectingSimplyProtectedWorksheet.php
<?php

namespace Aspose\Cells\WorkingWithWorksheets\SecurityFeatures;

use com\aspose\cells\Workbook as Workbook;
use com\aspose\cells\FileFormatType as FileFormatType;

class UnprotectingSimplyProtectedWorksheet {

    public static function run($dataDir=null)
    {

        $filesFormatType = new FileFormatType();

        //Instantiating a Workbook object
        $workbook = new Workbook($dataDir . "book1.xls");

        $worksheets = $workbook->getWorksheets();
        $worksheet = $worksheets->get(0);

        $protection = $worksheet->getProtection();

        //The following 3 methods are only for Excel 2000 and earlier formats
        $protection->setAllowEditingContent(false);
        $protection->setAllowEditingObject(false);
        $protection->setAllowEditingScenario(false);

        //Unprotecting the worksheet
        $worksheet->unprotect();

        // Save the excel file.
        $workbook->save($dataDir . "output.xls", $filesFormatType->EXCEL_97_TO_2003);

        //Print Message
        print "Worksheet unprotected successfully." . PHP_EOL;

    }

}

File: Plugins/Aspose_Cells_Java_for_Jython/asposecells/WorkingWithWorksheets/UnprotectingSimplyProtectedWorksheet.py
from asposecells import Settings
from com.aspose.cells import Workbook
from com.aspose.cells import SaveFormat
from com.aspose.cells import FileFormatType


class UnprotectingSimplyProtectedWorksheet:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWithWorksheets/UnprotectingSimplyProtectedWorksheet/'
        
        filesFormatType = FileFormatType

        #Instantiating a Workbook object
        workbook = Workbook(dataDir + "Book1.xls")

        worksheets = workbook.getWorksheets()
        worksheet = worksheets.get(0)

        protection = worksheet.getProtection()

        #The following 3 methods are only for Excel 2000 and earlier formats
        protection.setAllowEditingContent(False)
        protection.setAllowEditingObject(False)
        protection.setAllowEditingScenario(False)

        #Unprotecting the worksheet
        worksheet.unprotect()

        # Save the excel file.
        workbook.save(dataDir + "output.xls", filesFormatType.EXCEL_97_TO_2003)

        #Print Message
        print "Worksheet unprotected successfully."

if __name__ == '__main__':        
    UnprotectingSimplyProtectedWorksheet()